research/slim/datasets/mnist.py

```diff
@@ -15,7 +15,7 @@
 """Provides data for the MNIST dataset.
 
 The dataset scripts used to create the dataset can be found at:
-tensorflow/models/research/slim/datasets/download_and_convert_mnist.py
```

Comment:
Ditto (sorry for the repetition!).

Maybe do a grep for models/slim and find the remaining instances.